src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -772,11 +770,7 @@ export function submitAppointmentOrRequest(history) {
         });
         resetDataLayer();
 
-        if (featureVAOSServiceVAAppointments) {
-          history.push(`/va/${appointment.id}?confirmMsg=true`);
-        } else {
-          history.push('/new-appointment/confirmation');
-        }
+        history.push(`/va/${appointment.id}?confirmMsg=true`);
```

Comment:
Removing the `featureVAOSServiceVAAppointments` condition so v0 will take the same route as v2 does when submitting a direction schedule appointment.